streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -139,6 +139,8 @@
     private final static long DEFAULT_COMMIT_INTERVAL_MS = 30000L;
     private final static long EOS_DEFAULT_COMMIT_INTERVAL_MS = 100L;
 
+    public final static int DUMMY_THREAD_INDEX = 1;
```

Comment:
Since StreamsConfig is a public class, any public fields in this class would be exposed to users as well.

I'm wondering if we can just make it a public field inside `StreamThread` instead?

Reply:
Sounds good!